xmlfmt_test.go

```diff
@@ -79,26 +49,21 @@ x  <soapenv:Header/>
 x  <soapenv:Body>
 x   <ns:request>
 x    <ns:customer>
-x     <ns:id>123
-x     </ns:id>
-x     <ns:name type="NCHZ">John Brown
-x     </ns:name>
+x     <ns:id>123</ns:id>
```

Comment:
Better add a test case to show what would happen if the string in the middle (between the tags) is super long, more than several lines. What would the output like in that case.

thanks

Reply:
TestFormatXML_t5：TestFormatXML_t5